src/database/object/create-reference.ts

```diff
@@ -5,7 +5,7 @@ export function createObjectReference<T>(query: DatabaseQuery): AngularFireObjec
   return {
     query,
     snapshotChanges: createObjectSnapshotChanges(query),
-    update(data: Partial<T>) { return query.ref.update(data) as Promise<void>; },
+    update(data: Partial<T>) { return query.ref.update(data as any) as Promise<void>; },
```

Comment:
@davideast did this to quickly address `Argument of type 'Partial<T>' is not assignable to parameter of type 'Object'.` in Typescript 2.4